core/templates/dev/head/pages/collection_editor/collection_editor_navbar_directive.html

```diff
@@ -30,22 +30,18 @@
     </li>
 
     <!-- History -->
-    {% if SHOW_COLLECTION_NAVIGATION_TAB_HISTORY %}
-    <li ng-class="{'active': getTabStatuses().active === 'history'}">
+    <li ng-class="{'active': getTabStatuses().active === 'history'}" ng-if="SHOW_COLLECTION_NAVIGATION_TAB_HISTORY">
```

Comment:
Actually -- could we just delete the code to show these two tabs (history and stats) altogether? I don't think it's going to happen for quite a while.